src/components/HotTopics/HotTopics.css

```diff
@@ -0,0 +1,30 @@
+.hot-topics {
+  height: 85vh;
+  background-color: #F6F9FC;
+  padding-top: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+.hot-topics__desc {
+  padding: 2rem;
+  text-align: center;
+}
+.hot-topics__buttons {
```

Comment:
nice work getting grid going even though you didn't feel 100% confident on it! -J